source/variables/colors.css

```diff
@@ -0,0 +1,14 @@
+/* 
+The color palette for the project.
+http://cssnext.io/features/#color-function 
+
+background-color: var(--primary);
+color: var(--greyDark);
+*/
+:root {
+	--primary: #e55030;
+	--secondary: #d1d2d4;
+	--grey-light: #eeeeee;
```

Comment:
I never understood it either, especially when we had the lighten and darken color functions.
Agree, I can trim this down to two colors and not make them monochromatic colors :)